brickstrap.sh

```diff
@@ -214,9 +238,10 @@ function brp_sanity_check_cli_options()
 
 function brp_validate_cli_options()
 {
-    brp_sanity_check_cli_options "$@" && \
-    brp_validate_project_name && \
+    brp_sanity_check_cli_options "$@"
+    brp_validate_project_name
     brp_validate_component_names
+    brp_validate_qemu || [ $? -eq 255 ] # no QEMU specified = 255
```

Comment:
I know that you can't run `update-binfmts --find`, but you can run `update-binfmts --version` to make sure that `update-binfmts` itself exists.